docs/supervisor.md

```diff
@@ -14,50 +15,37 @@
 |:---------------:|:----------------------------------------------------------------------:|:------------------:|:--------------:|:---------------------------------:|
 |  vSphere 7.0 U1c/P02 + ESXi 7.0 U1c/P02 | Bundled with vSphere |     Bundled with vSphere (v1.16-v1.19)    |     v1.5.1     |         v1.1.0 and higher         |
 
-
 ## Prerequisites
 
 * [Install Data Manager](supervisor-datamgr.md)
 * The following vSphere privileges are required if users do not have an Administrator role in vSphere.
-    * SupervisorServices.Manage
-    * Namespaces.Manage
-    * Namespaces.Configure
- 
+  * SupervisorServices.Manage
+  * Namespaces.Manage
+  * Namespaces.Configure
 
 ## Install
 
-In vSphere with Kubernetes Supervisor cluster, users are supposed to leverage `Velero vSphere Operator` to install velero
-as well as velero-plugin-for-vsphere, since vSphere users who install velero don't have cluster-admin role in Supervisor
-cluster. Please refer to
+In a **vSphere with Tanzu** Supervisor cluster, users are supposed to leverage `Velero vSphere Operator` to install velero as well as velero-plugin-for-vsphere, since vSphere users who install velero don't have cluster-admin role in Supervisor cluster. Please refer to
```

Comment:
Should "for the detail" on line 30 be removed?

Reply:
Removed.